packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/incremental/IncrementalLayoutDiagramConverter.java

```diff
@@ -132,19 +140,106 @@ private EdgeLayoutData convertEdge(Edge edge, Map<String, ILayoutData> id2Layout
         return layoutData;
     }
 
-    private LabelLayoutData convertLabel(Label label, Map<String, ILayoutData> id2LayoutData) {
+    private LabelLayoutData convertEdgeLabel(Edge edge, Label label, LabelType labelType, Map<String, ILayoutData> id2LayoutData) {
```

Comment:
The `Edge edge` parameter is not actually ussed.

Reply:
Done